tests/queries/0_stateless/01748_partition_id_pruning.sql

```diff
@@ -0,0 +1,19 @@
+drop table if exists x;
+
+create table x (i int, j int) engine MergeTree partition by i order by j settings index_granularity = 1;
+
+insert into x values (1, 1), (1, 2), (1, 3), (2, 4), (2, 5), (2, 6);
+
+set max_rows_to_read = 3;
+
+select * from x where _partition_id = partitionID(1);
```

Comment:
We have function `buildId` where the `Id` is written as `Id`.
It will be more consistent to also write `partitionId`.